### ISSUE TITLE
Give trees and lists accessible names

### ISSUE THREAD

**Issue by vkbo** [vkbo/novelWriter#2106]
All tree and list views in the app should have the accessibility name set to the text content of their labels. It doesn't look like setting them as a label "buddy" is enough.
